.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Integation tests
```

Comment:
Why do we need separate workflow for tests? We already have integration tests enabled in primary workflow

Reply:
@g4s8 It's because it could take so much time to be executed 6 times

Reply:
For each of them, we have to pull docker image and run all integration tests

Reply:
So, by my configuration, I run only unit tests in primary workflow and integration tests in `integration-test.yml`

Reply:
You could check it in build logs.

Reply:
@g4s8 WDYT ?

Reply:
@g4s8 On reflection, I think I will use primary workflow to enable `testcontainers` for integration tests. It will allow us to test in several environments (OS and JDK).

Reply:
I don't think it's a good reasons for disabling it. GitHub pulls images quite fast. Please revert workflows back to check etcd rest cases in standard `verify` workflow

Reply:
@g4s8 I agree with you